scripts/xGDB_Procedure.sh

```diff
@@ -193,16 +193,12 @@ FirstPart() {
       msg_blast="${space}${count2} $name sequences copied from $download_destination to $blast_destination (2.${step}5)"
       echo "$msg_blast"  >>$WorkDIR/logs/Pipeline_procedure.log
 
-      declare $type_count=$count2 # The value of this variable variable (e.g. est_count, etc) is used later for sanity check on parsing and upload
-      eval \${type_count}=${count2} # force evaluation
-#debug only     echo "${space}$est_count, $cdna_count, $tsa_count, $prot_count, $gdna_count, $gdnarm_count, $annotpep_count, $annotmrna_count, $annotcds_count, $cpgatpep_count, $cpgatmrna_count, $cpgatcds_count" >>$WorkDIR/logs/Pipeline_procedure.log
+      declare $type_count=$count2 # The value of this variable (est_count, etc.) is used later for sanity check on parsing and upload
```

Comment:
I would still contend, as I did in email, that `eval` is more intuitive than `declare` in this case, and (in the current usage) functionally equivalent. Not a critical issue though.